models/server-models.js

```diff
@@ -1,10 +1,25 @@
 const db = require("../db/connection");
 
-exports.selectTopics = (req, res, next) => {
+exports.selectTopics = () => {
     let queryString =
     "SELECT * FROM topics;";
 
     return db.query(queryString).then((result) => {
         return result.rows;
       });
+}
+
+exports.selectArticleById= (article_id) => {
+    let queryString =
```

Comment:
Is this query string being changed at any point? If not, this should be declared with const.